data/relex-semantic.algs

```diff
@@ -331,6 +332,12 @@ DIR_OBJ1_1
 <F_L obj> = %
 <F_L obj> = <F_R linkR0 F_R ref>
 
+#TemplateActionAlg
+DIROBJ_Osm
+<LAB> = \O\.*\.m

```

Comment:
When adding new rules, its great to add one or two example sentences that trigger these rules